src/Sulu/Bundle/PageBundle/Controller/PageController.php

```diff
@@ -621,10 +621,12 @@ private function getWebspaceNodes(
         $webspaces = [];
         /** @var Webspace $webspace */
         foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
-            if (null === $webspace->getLocalization($locale)) {
+            if (null === $webspace->getLocalization($locale) || false === $this->securityChecker->hasPermission(
+                new SecurityCondition('sulu.webspaces.' . $webspace->getKey(), $locale, SecurityBehavior::class),
```

Comment:
Use `PageAdmin::getPageSecurityContext($webspace->getKey())` instead of `'sulu.webspaces' . $webspace->getKey()`

Reply:
This does not exist on `2.1` but will port it back to `2.1` branch so we can also use it there.

Reply:
PR updated!